Prace_domowe/Praca_domowa1/Grupa2/Paweł_Fijałkowski/pd1.ipynb

```diff
@@ -0,0 +1,719 @@
+{
```

Comment:
<p>Szkoda że to nie jest wykres -- jak jest tak dużo liczb, to trzeba poświęcić dużo czasu żeby znaleźć tu silnie skorelowane zmienne. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>